elearning-backend/app/Http/Requests/UpdateCategoriesPostRequest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class UpdateCategoriesPostRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            // 'title' => [
+            //     'required',
+            //     'max:100',
+            //     Rule::unique("categories", "title")
+            // ],
+            'description' => [
+                'required',
+                'max:200'
+            ],
+            // 'slug' => [
+            //     'required',
+            //     Rule::unique("categories", "slug")
+            // ]
+            // 'title' => [],
+            // 'slug' => []
+        ];
```

Comment:
remove commented lines if not used anymore

Reply:
Comments removed @chevyB

Reply:
Comments have been removed.